### ISSUE TITLE
Plantuml Beta version features are not able to integrate with doxygen.

### ISSUE THREAD

**Issue by SaravanaRaja88** [doxygen/doxygen#7707]
For example latest feature with plantuml is Mindmap as below.

```
@startmindmap
* Debian
** Ubuntu
*** Linux Mint
@endmindmap
```

This will generate some diagram,  So i am not able to generate the diagram with doxygen.

**Comment by albert-github** [comment]
See my comment at https://forum.plantuml.net/11230/doxyen-not-able-to-generate-the-mindmap-feature-and-wbs. Just trying to find the source of the problem.

**Comment by albert-github** [comment]
By means of #8478 doxygen's plantuml capabilities have been slightly extended and it is now possible to create mindmaps as well.
Doxygen does not directly support the command `@startmindmap` but supports ``statuml{mindmap}` and by means of `ALIASES` this can be mad transparent.
Example
```
/// \file
/// @startmindmap
/// * Debian
/// ** Ubuntu
/// *** Linux Mint
/// @endmindmap
///
```
and the settings:
```
QUIET=YES
ALIASES += startmindmap=@startuml{mindmap}
ALIASES += endmindmap=@enduml
PLANTUML_JAR_PATH      = plantuml.jar
```

the required functionality is available.

Example: [example.tar.gz](https://github.com/doxygen/doxygen/files/6680615/example.tar.gz)

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.2.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
